docs/ledger.md

```diff
@@ -0,0 +1,115 @@
+---
+id: ledger
+title: Ledger
+---
+
+# Ledger vault
+
+Connect the ledger device to the system in which signatory is running.
+Install tezos-wallet and tezos-baker apps from [ledger live](https://www.ledger.com/ledger-live/download).
+
+Note: Developer mode might be needed to install baker app.
+[Ledger Developer mode](https://developers.ledger.com/docs/live-app/developer-mode/#:~:text=To%20activate%20the%20Developer%20mode,Live%20version%202.32%20and%20above.)
+
+## Configuration
```

Comment:
Are these all required? We may want to include the required column just to keep consistent with the other pages